CMakeLists.txt

```diff
@@ -1199,7 +1199,7 @@ math(EXPR VERSION_NUMBER_INTEGER
 # 5.0 broke backcompat in drsyms and xmm opnd sizes
 # 4.1 broke backcompat in drsyms + 64-bit core (opcodes + reachability)
 # 4.0 broke backcompat in drmgr, drsyms, drinjectlib, and dr_get_milliseconds()
-set(OLDEST_COMPATIBLE_VERSION_DEFAULT "701")
+set(OLDEST_COMPATIBLE_VERSION_DEFAULT "790")
```

Comment:
BTW, did you take a look at make/DynamoRIOConfigVersion.cmake.in?  I don't think it does the right thing: someone asking for 7.1 will be handed 7.90 with a claim that it's compatible.  Maybe just an issue should be filed as it's a separate pre-existing problem.

Reply:
Filed https://github.com/DynamoRIO/dynamorio/issues/3533.